tests/test_multiplayer_login.py

```
@@ -76,3 +76,172 @@ def test_close_button_returns_back(self):
+
+        with patch.object(login_mod, "pygame", mock_pg), \
+             patch.object(login_mod, "animation_utils"), \
+             patch.object(login_mod, "Button", return_value=Mock(handle_event=Mock(return_value=False))):
+            mock_pg.event.get.return_value = [esc_ev]
+            screen = login_mod.MultiplayerLoginScreen(mock_screen, mock_client)
+            result = run_async(screen.run())
+
+        assert result == "back"
+
+    def test_backspace_removes_character(self):
+        screen, mock_pg, mock_client = _make_login_screen()
```

Ruff (0.15.12)

[warning] Unpacked variable `mock_pg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)